test/inc/TestDef.h

```diff
@@ -15,9 +15,14 @@ static const std::wstring c_testFileExtension_Packaged = L".reuniontestfile-pack
 static const std::wstring c_testFilePhaseEventName = L"ReunionTestFilePhaseEventName";
 
 static const std::wstring c_testStartupPhaseEventName = L"ReunionTestStartupPhaseEventName";
-
 static const std::wstring c_testInstanceRedirectedPhaseEventName = L"ReunionTestInstanceRedirectedPhaseEventName";
 
+inline const winrt::hstring c_rawNotificationPayload = L"<toast></toast>";
+inline IID c_comServerId = winrt::guid("ccd2ae3f-764f-4ae3-be45-9804761b28b2"); // Value from PushNotificationsTestAppPackage ComActivator in appxmanifest.
+inline IID c_fakeComServerId = winrt::guid("00000000-0000-0000-0000-000000000001");
+inline const int c_pushTestWait = 3000;
+inline const int c_pushTestChannelWait = c_pushTestWait + 300000; // Need to wait 300000ms for channel request + 3000ms for application overhead 
```

Comment:
These are PushNotification specific definitions
And lines above are AppLifecycle specific definitions?

If so we'll be better served splitting these into feature-specific files, e.g. PushNotificationTestConstants.h and AppLifecycleTestConstants.h.

Just think if a year from now we decide PushNotifications (or AppLifecycle or...) should relocate. Figuring out which albums to take vs which stay can be irksome ;-)

If there really are shared constants then yes, a common header makes sense (though something less generically named is desirable). But I'm doubtful features really have many shared constants.

@aeloros should be refactored in a follow up PR?

Reply:
The first two need to be shared so our TestApp has access to those constants. The other two can be moved to APITests.cpp.